modules/mobile/app/controllers/mobile/discovery_controller.rb

```diff
@@ -6,10 +6,164 @@ module Mobile
   class DiscoveryController < ApplicationController
     skip_before_action :authenticate
 
-    def index
-      # TODO: this will be the endpoint that returns the discovery service map
-      # for now it returns a simple welcome message
+    API_VERSION_MAP = {
+      deprecated: {
+        min_build_number: {
+          ios: 0,
+          android: 0
+        },
+        max_build_number: {
+          ios: 19,
+          android: 19
+        },
+        api_version: 'deprecated',
+        webviews: {
+        },
+        endpoints: {
+        },
+        display_message: 'Please update the app.',
+        app_access: false
+      },
+      one_zero: {
+        min_build_number: {
+          ios: 20,
+          android: 20
+        },
+        max_build_number: {
+          ios: 999,
+          android: 999
+        },
+        api_version: '1.0',
+        webviews: {
+          corona_FAQ: 'https://www.va.gov/coronavirus-veteran-frequently-asked-questions',
+          facility_locator: 'https://www.va.gov/find-locations/'
+        },
+        endpoints: {
+          appeal_details: {
+            url: '/appeal/:id',
+            verb: 'GET'
```

Comment:
Is it useful to respond with the verb, or will a mobile app already be coded to be a GET/POST/PUT?

Reply:
Not sure yet, waiting to see what Patrick thinks

Reply:
Patrick said the verbs would be useful for the frontend to have. Let me know if we should all get together to discuss that more.